<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogRename</class>
 <widget class="QDialog" name="DialogRename">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="2">
    <widget class="QPushButton" name="pbOk">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="hs">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pbCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lRename">
        <property name="text">
         <string>Rename</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="leNewFilename"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
